blockstore/splitstore/splitstore_gc.go

```diff
@@ -7,20 +7,27 @@ import (
 )
 
 func (s *SplitStore) gcHotstore() {
+	// we only perform moving gc every 10 compactions as it can take a while
+	if s.compactionIndex%10 != 0 {
+		goto online_gc
```

Comment:
note for steb: this goto is very short-lived; it disappears a couple of commits later.